src/FirstSteps/RPG/Heroes/Human.cs

```diff
@@ -4,27 +4,42 @@ namespace FirstSteps.RPG.Heroes
 {
     public class Human : Hero
     {
+        public int CoinsBag { get; set; }
```

Comment:
`private int _coinsBag;` is an element of `Hero.cs`. You can change the protection level instead of adding doubled properties.